content/wiki/autotest-integration.md

```diff
@@ -54,7 +55,7 @@ autotest-all: --color
 
 Please see [Running Features](/cucumber/running-features/) for more information about profiles, and the [Ruby on Rails](/implementations/ruby/ruby-on-rails/) page for extra help when running in that context.
 
-If you get an error like `When using several—format options only one of them can be without a file (RuntimeError)` when running features with autospec, you are probably accidentally requiring 'spec'. Read about the solution on Troubleshooting.
+If you get an error like `When using several—format options only one of them can be without a file (RuntimeError)` when running Features with autospec, you are probably accidentally requiring `spec`. Read about the solution on [[Troubleshooting]].
```

Comment:
[[]] for "troubleshooting should be removed? (or a link provided).
(I thought I had already removed this link. Are you working of the latest master?)